src/cephadm/cephadm

```diff
@@ -5237,6 +5245,7 @@ def command_deploy(ctx):
 
         c = get_container(ctx, ctx.fsid, daemon_type, daemon_id,
                           ptrace=ctx.allow_ptrace)
+        c = apply_extra_container_args(ctx, c)
```

Comment:
Since this call appears to always follow `get_container` would it make sense to have a:
```python
# pseudocode
def get_container_with_extra_args(ctx, ...):
   return apply_extra_container_args(ctx, get_container(ctx, ...))
```
Rather than repeat a call to apply_extra_container_args  in many locations?